src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+public void doWork() {
+    System.out.println("Truck do work");
+}
+@Override
+public void stopWork() {
+    System.out.println("Truck stoped work");
```

Comment:
There is a typo in the string "Truck stoped work". It should be "Truck stopped work".